Description: Bump constant_time_eq dependency
Author: Jochen Sprickerhof <jspricke@debian.org>
Origin: vendor
Forwarded: no
Last-Update: 2023-08-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: blake2s-simd/Cargo.toml
===================================================================
--- blake2s-simd.orig/Cargo.toml
+++ blake2s-simd/Cargo.toml
@@ -33,7 +33,7 @@ version = "0.7.0"
 default-features = false
 
 [dependencies.constant_time_eq]
-version = "0.2.4"
+version = "0.1"
 
 [features]
 default = ["std"]
